app/controllers/passwordless/sessions_controller.rb

```diff
@@ -5,6 +5,8 @@
 module Passwordless
   # Controller for managing Passwordless sessions
   class SessionsController < ApplicationController
+    # Raise this exception when a session is expired.
+    class ExpiredSessionError < StandardError; end
```

Comment:
Add a new line under this and then we’re good to go!

Reply:
Done